pyproject.toml

```diff
@@ -132,7 +133,7 @@ lint.fixable = [
 # W191 - tab-indentation*
 # PGH003 - blanket-type-ignore
 # *ignored for compatibility with formatter
-lint.ignore = [
```

Comment:
Did you not get warnings after removing this?

Reply:
It's a minor change, but in the template (https://github.com/GenomicMedLab/software-templates/blob/05c35343641086045f3c6205a2ebe7524f28d1be/python/%7B%7Bcookiecutter.project_slug%7D%7D/pyproject.toml#L69) I just have a separate primary TOML section for `tool.ruff.lint` rather than making each of the `lint` subsections live under `tool.ruff`

Reply:
(line 76 being where the new section is added here)

Reply:
Oh, I didn't see line 76